src/shed/_codemods.py

```diff
@@ -176,6 +176,38 @@ def remove_pointless_parens_around_call(self, _, updated_node):
         except SyntaxError:
             return updated_node
 
+    @m.leave(
+        m.Call(func=m.Name("dict"), args=[])
+        | m.Call(func=m.Name("list"), args=[])
+        | m.Call(func=m.Name("tuple"), args=[])
+    )
+    def replace_builtin_with_literal(self, _, updated_node):
+        if updated_node.func.value == "dict":
+            return cst.Dict([])
+        elif updated_node.func.value == "list":
+            return cst.List([])
+        else:
+            return cst.Tuple([])
```

Comment:
I like to include an assertion here to make sure I'm handling the expected value.